modules/serverless-pingsource-odc.adoc

```diff
@@ -39,9 +39,17 @@ spec:
 .. Click *Create*.
 
 . Create a PingSource in the same namespace as the service created in the previous step, or any other sink that you want to send events to.
-.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*.
-.. Select *Ping Source*.
-.. Enter a value for *Schedule*. In this example, the value is `*/2 * * * *`, which creates a PingSource that sends a message every two minutes.
+.. In the *Developer* perspective, navigate to *+Add* -> *Event Source*. The  *Event Sources* page in the *Developer Catalog* is displayed.
++
+If you have multiple providers, you can filter the available event sources based on the *Provider*.
+.. In the *Event Sources* page, select *Ping Source* and click *Create Event Source*. The *Create Event Source* page is displayed.
++
+[NOTE]
+====
+You can configure the *PingSource* settings by using the *Form view* or *YAML view*.
+Where available, you can switch between the *Form view* and *YAML view*. The data is persisted when switching between the views.
+====
+.. In the *Create Event Source* page, enter a value for *Schedule*. In this example, the value is `*/2 * * * *`, which creates a PingSource that sends a message every two minutes.
```

Comment:
Remove 'In the *Create Event Source* page,'

Reply:
Done.